### DISCUSSION TITLE
极空间 nas 部署报错，请问怎么解决，感谢🙏

### DISCUSSION THREAD

**Discussion by zealiao** [hslr-s/sun-panel#253] [General]
![image](https://github.com/user-attachments/assets/34ec0961-574f-42cb-9130-13545b4bae20)
看了群晖的解决方案，但是不知道怎么在极空间里操作，另外极空间好像还没开放 SSH，所以有没有极空间部署成功的大佬可以指点一下？

**Comment by anan0471** [comment]
一样的问题，不知道怎么弄。

**Reply by zealiao** [reply]
没部署成功我就已经买了，尴尬……

**Comment by hslr-s** [comment]
请尝试参考 https://github.com/hslr-s/sun-panel/discussions/255
